### PR TITLE
Add Asset Helpers to Default Layout

### PR DESCRIPTION
**Pull request by josiahsprague**
Add Asset Helpers
- [x] scripts
- [x] styles

### CODE REVIEW

**Comment by josiahsprague** [comment]
This is ready for review.

File: defaults/scripts.hbs [hunk 1]
```diff
@@ -1 +1,4 @@
 <script src="/compiled/scripts.js"></script>
```
Comment (joanniclaborde) [inline defaults/scripts.hbs]: Should the compiled scripts go after the jungle scripts? So they can override / customize the default scripts?

Comment (pushred) [inline defaults/scripts.hbs]: I agree, the compiled scripts will be depending upon the jQuery included in the default scripts as well.

Comment (josiahsprague) [inline defaults/scripts.hbs]: No. The compiled scripts are the default site bundle. They need to go first. That's the order they were in in CliqueTools. These are migrated sites, so most of them won't be overriding anything new. But if something is over-ridden, either a new page/section script can be added to site.yml, or the over-riding scripts can be added to the end of the bundle.

Comment (pushred) [inline defaults/scripts.hbs]: Oh yeah, I forgot the ordering of the _browserified_ stuff is controlled in the task.

**Comment by krackydile** [comment]
Do section specific styles load correctly for you @josiahsprague? I'm not sure why but for me they are retaining the `/assets` in the route. As an example:

```<link rel="stylesheet" href="./assets/styles/section-news.css" />```

**Comment by josiahsprague** [comment]
Pushed a fix to this branch @krackydile

**Pull request merged**